independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-base/java/runtime/src/main/resources/META-INF/quarkus-extension.tpl.qute.yaml

```diff
@@ -1,9 +1,9 @@
 name: {extension.name}
-#description: {extension.name} ...
+#description: {extension.description ?: 'Do something useful.'}
 metadata:
 #  keywords:
 #    - {extension.id}
-#  guide: ...
+#  guide: {extension.guide ?: '...'}
```

Comment:
No CLI option for this? later?

Reply:
Well, I don't think people will actually know the URL when creating the extension. So I just generate one that is consistent with our conventions for the Quarkiverse.